src/content/docs/en/welcome.mdx

```diff
@@ -124,21 +124,21 @@ However, Ultramarine Linux includes Terra, RPM Fusion and Flathub.
 ### Why Ultramarine (instead of Windows)
 
 - Performance
-  - Windows bloats your computer with many different potentially unwanted programs that slow down your computer. Ultramarine does not.
-  - Ultramarine provides a few lightweight desktop environments (namely Budgie/Flagship and XFCE), which run faster and provide a snappier experience.
-  - Ultramarine uses newer technologies that are not available on Windows because of backwards compatibility. These technologies brings better performance.
+  - Windows ships many different potentially unwanted programs that slow down your computer. Ultramarine does not.
+  - Ultramarine provides a few lightweight desktop environments (namely Budgie/Flagship and XFCE) that offer more control to the average user without harming performance and usability.
+  - Ultramarine uses newer technologies that are not available on Windows because of backwards compatibility. These technologies bring better performance.
     - For example, task schedulers, file systems, shells, etc.
     - Most modern servers (including giant web servers) use Linux simply because it's much faster.
 - Privacy
-  - We do not collect/track your private data ever because that's a waste of storage.
-  - We don't even know how many Ultramarine devices there are (unlike some companies).
-  - When you setup Windows, you need to accept a very long license. Well, we don't even have a license agreement page in our installer.
-  - Ultramarine is open source. Anyone can check if we track/collect your data by reviewing the source code. We don't.
+  - We do not collect/track personally identifiable information. We have no use for it.
+  - We have no way of tracking how many people use Ultramarine.

```

Comment:
this isn't necessarily true and i'm not sure how it got in in the first place, dnf counting allows us to get a pretty reliable count of UM systems grabbing stuff from Fedora and us